src/GeneralStateTestsFiller/stEWASMTests/initMalformedBytecodeHasStartFromTxFiller.yml

```diff
@@ -12,7 +12,7 @@ initMalformedBytecodeHasStartFromTx:
     a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
       balance: '100000000000'
       code: ''
-      nonce: ''
+      nonce: '0'
```

Comment:
This isn't needed in other fillers. Which one is correct? With `'0'` or `''`?

Reply:
Both are correct, it works with `'0'` and with `''`, this was the first test I changed, then I changed the others and realized the real problem was in field `shouldnotexist`